Support spring 2.4.3

<!--- This template is for bugs. Remove it for other issues. -->


### Description
When using BP with Spring Boot to 2.4.3+ (works with 2.4.2) an exception is thrown.

### Expected behavior
No exception :)

### Actual behavior
```
  Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestMappingHandlerAdapter' defined in class path resource [org/springframework/boot/autoconfigure/web/servlet/WebMvcAutoConfiguration$EnableWebMvcConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter]: Factory method 'requestMappingHandlerAdapter' threw exception; nested exception is java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper

  Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter]: Factory method 'requestMappingHandlerAdapter' threw exception; nested exception is java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper

  Caused by: java.lang.NoSuchFieldError: com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.objectMapper
```
### Steps to reproduce

### Environment
<!--- Environment info like e.g.					-->
<!--- Version:            			-->
<!--- JPA-Provider:       Hibernate 5.2.7.Final		-->
<!--- DBMS:               PostgreSQL 9.6.2			-->
<!--- Application Server: Java SE					-->

Version:  1.6.0-Alpha2	           
JPA-Provider: Hibernate 
DBMS:  Mariadb 10.3

--- a/integration/spring-data/webmvc/src/main/java/com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.java
+++ b/integration/spring-data/webmvc/src/main/java/com/blazebit/persistence/spring/data/webmvc/impl/json/EntityViewAwareMappingJackson2HttpMessageConverter.java
@@ -38,7 +38,7 @@ public class EntityViewAwareMappingJackson2HttpMessageConverter extends MappingJ
     private final EntityViewAwareObjectMapper entityViewAwareObjectMapper;
 
     public EntityViewAwareMappingJackson2HttpMessageConverter(final EntityViewManager entityViewManager, EntityViewIdValueAccessor entityViewIdValueAccessor) {
-        this.entityViewAwareObjectMapper = new EntityViewAwareObjectMapper(entityViewManager, objectMapper, entityViewIdValueAccessor);
+        this.entityViewAwareObjectMapper = new EntityViewAwareObjectMapper(entityViewManager, getObjectMapper(), entityViewIdValueAccessor);
     }
 
     @Override
@@ -79,7 +79,7 @@ public class EntityViewAwareMappingJackson2HttpMessageConverter extends MappingJ
             if (inputMessage instanceof MappingJacksonInputMessage) {
                 Class<?> deserializationView = ((MappingJacksonInputMessage) inputMessage).getDeserializationView();
                 if (deserializationView != null) {
-                    return this.objectMapper.readerWithView(deserializationView).forType(javaType).
+                    return this.getObjectMapper().readerWithView(deserializationView).forType(javaType).
                             readValue(inputMessage.getBody());
                 }
             }

